api/pkg/apis/v1alpha1/providers/states/k8s/k8s.go

```diff
@@ -525,183 +528,3 @@ func (s *K8sStateProvider) Get(ctx context.Context, request states.GetRequest) (
 	}
 	return ret, nil
 }
-
```

Comment:
After these incompatible methods are removed, shouldn't we add compatible implementations?

Reply:
We already use config manager to read values in catalogs. I think these codes are duplicate and legacy.